pkg/rtc/participant.go

```diff
@@ -666,24 +667,39 @@ func (p *ParticipantImpl) GetAudioLevel() (level uint8, active bool) {
 	return
 }
 
-func (p *ParticipantImpl) GetConnectionQuality() livekit.ConnectionQuality {
+func (p *ParticipantImpl) GetConnectionQuality() *livekit.ConnectionQualityInfo {
 	// avg loss across all tracks, weigh published the same as subscribed
 	var pubLoss, subLoss uint32
 	var reducedQualityPub bool
 	var reducedQualitySub bool
+	var pubAudioScore float64
+	var numAudioTracks int
 	p.lock.RLock()
 	defer p.lock.RUnlock()
 	for _, pubTrack := range p.publishedTracks {
 		if pubTrack.IsMuted() {
 			continue
 		}
-		pubLoss += pubTrack.PublishLossPercentage()
+		// audio has scores available
+		if pubTrack.Kind() == livekit.TrackType_AUDIO {
+			pubAudioScore += pubTrack.GetUpConnectionScore()
+			numAudioTracks++
+		} else {
+			pubLoss += pubTrack.PublishLossPercentage()
+		}
 		publishing, registered := pubTrack.NumUpTracks()
```

Comment:
this would be for video only, if we are keeping the old algorithm, for audio it should be `1, 1`